### DISCUSSION TITLE
Add dictionary support

### DISCUSSION THREAD

**Discussion by ash1247** [TheRakiburKhan/RKAPIService#2] [Ideas]
Please add dictionary support to body. As AF supports it, most of the developers use dictionary to send body params

**Comment by TheRakiburKhan** [comment]
Thanks for the idea, we will introduce it in next update!

**Reply by TheRakiburKhan** [reply]
> Please add dictionary support to body. As AF supports it, most of the developers use dictionary to send body params

We have added this feature on 2.5.0 release. Thank you for your idea. Stay with us.
